modules/ROOT/examples/swift/code-snippets/SampleCodeTest.swift

```diff
@@ -706,6 +707,45 @@ class SampleCodeTest {
         replicator.start()
         // end::getting-started[]
     }
+    
+    func dontTestPredictionQuery() throws {
+        let database: Database
+        do {
+            database = try Database(name: "mydb")
+        } catch {
+            fatalError("Error opening database")
+        }
+        
+        // tag::predictive-query[]
+        // Load MLModel from `Sentiment.mlmodel`
+        let modelURL = Bundle.main.url(forResource: "Sentiment", withExtension: "mlmodel")!
+        let compiledModelURL = try MLModel.compileModel(at: modelURL)
+        let model = try MLModel(contentsOf: compiledModelURL)
+        let predictiveModel = CoreMLPredictiveModel(mlModel: model)
```

Comment:
After covering the abstract interface "PreditiveModel", You can add CoreMLPredictiveModel as a sub-section (only for iOS)
- [ ]  you can discuss the CoreMLPredictiveModel concrete implementation using same model example. Difference is that you don't implement the PredictiveModel interface 
- [ ] registration, reregistration and indexing is same as in previous abstract interface case. So no need to repeat
- [ ] Mention the outputTransform function to transform data output so it can be directly used in our queries (the dynamic keys use case in classifiers)